cni/pkg/cmd/root.go

```diff
@@ -133,11 +133,7 @@ var rootCmd = &cobra.Command{
 				// new ambient-enabled pods while our replacement spins up.
 				if !isUpgrade {
 					if cleanErr := installer.Cleanup(); cleanErr != nil {
-						if err != nil {
-							err = fmt.Errorf("%s: %w", cleanErr.Error(), err)
-						} else {
-							err = cleanErr
-						}
+						fmt.Println(cleanErr.Error())
```

Comment:
println instead of log? is that intended?

Reply:
My reflex is to avoid relying on logging frameworks in top-level `defer` funcs since I don't know what state the logging framework is in, but I think it's probably fine in this  particular case.

Reply:
In a panic its plausibly needed (not sure) but we should be guaranteed in this case AFAIK